posix.mak

```diff
@@ -449,7 +450,7 @@ rsync : all kindle pdf
 rsync-only :
 	rsync -avzO --chmod=u=rwX,g=rwX,o=rX --delete $(RSYNC_FILTER) $W/ $(REMOTE_DIR)/
 
-dautotest: all verbatim pdf diffable-intermediaries
+dautotest: all verbatim pdf diffable-intermediaries ${DUB_DIR}
```

Comment:
Is this the right place to add the target? Wouldn't the target that builds the changelog be more suitable?

Reply:
Ok